aten/src/ATen/native/transformers/cuda/attention.cu

```diff
@@ -387,7 +387,7 @@ __host__ std::tuple<Tensor, Tensor, Tensor> transform_bias_rescale_qkv_cuda(
           auto sizes = collapse_dims_1_and_2(nt_qkv->get_nested_size_tensor());
           auto offsets =
               NestedTensor_batch_offsets_from_size_tensor(sizes, sizes.numel());
-          at::native::narrow(offsets, 0, sizes.numel() + 1, sizes.numel())
+          at::native::narrow_symint(offsets, 0, sizes.numel() + 1, sizes.numel())
```

Comment:
You may want to do the same thing we did with reshape() and keep a non symint native::narrow around so you don't have to go update all these call sites

Reply:
is the non symint native::narrow just declared manually in a non codegen header or is there a trick (i couldn't find anything looking at native_functions.yaml for reshape)

Reply:
ok i tried this but i got undefined narrow in libtorch_cuda.so, when I included BC header in `aten/src/ATen/native/transformers/cuda/attention.cu`. I guess there is probably something else I need to change about the build to make this work?

Reply:
i gave up on this part bc its just a couple callsites. could revisit if needed.

Reply:
Did you put TORCH_API on the native:: header, and properly include it at the definition site?